lib/util/dynamicimport.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
```

Comment:
I'm not sure if there should be a note here for us to come back around to fix this in the future? Might be useful for future us as a reminder.

Reply:
I've added a note above the line where we create the function.